src/components/LayerControl.vue

```diff
@@ -66,6 +78,9 @@ export default {
       }
       this.changeLayer(id);
     },
+    cloneLayer() {
+      this.$store.commit('keymap/duplicateLayer', { layer: this.layer });
```

Comment:
you should be able to use the vuex helpers, and simplify this to `this.duplicateLayer({layer: this.layer})`